### ISSUE TITLE
Deploy goes through, but registers as failed in the Cloud dashboard

### ISSUE THREAD

**Issue by RasmusUmbraco** [umbraco/Umbraco.Cloud.Issues#553]
### Issue description

Umbraco 8.13.1
Forms 8.7.6
Deploy 4.3.0.0

When deploying between environments the deployments show as failed in the Cloud dashboard
![image](https://user-images.githubusercontent.com/88326257/156194815-83ce406c-03e7-48a0-9933-752c20e0c7cd.png)
But changes went through in the backoffice
![image](https://user-images.githubusercontent.com/88326257/156194928-42ea6ef7-2bd0-4f5d-83ea-9501938af862.png)
The deployment also shows as succeeded
![image](https://user-images.githubusercontent.com/88326257/156195225-bd0b00f4-535b-4f6a-94cc-cf6abc1c40ab.png)
The changes are also comitted directly to git.
Eric suggested that the Site Extension may not be getting the proper data from the portal and he verified that there are no errors logged in the log files of azure application insight.

**Comment by sajumb** [comment]
@RasmusUmbraco I have not been able to reproduce this on any of the projects that I have considered. Please send me more details and we will consider it as soon as possible.

**Comment by rhealefever** [comment]
We run into this issue fairly reliably every time we run the Cloud Developer training course. I just ran into it during a demonstration.

Umbraco 9.4.3
Forms 9.3.0
Deploy 9.3.1

On my Dev env, I created a couple of doctypes, some templates, and some content. Deploying/transferring those changes to Live is initially successful.

On my Dev env, I modify a doctype by adding a new property, and deploy to Live. Live fails on data extraction.

![image](https://user-images.githubusercontent.com/47108275/165256541-56f27ab6-38c5-42d3-822a-dfcc9792600d.png)

Since the changes are there in the repo, I can get the changes by triggering a data extraction from the backoffice and then refresh the page, and then I see the new property on the Live env. But the error persists on the Live env.

A couple of project aliases this happened on:
cloud-ral-umbraco-dk
cloud-test
